portfolio/src/main/webapp/script.js

```diff
@@ -27,8 +27,112 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-const ham = document.querySelector('.ham')
+//var myBody = getElementByTagName('body');
 
-ham.addEventlisteber('click', function(){
-    this.classList.toggle('is-active');
-})
\ No newline at end of file
+
+// translating text in testing phase
+/*
+function requestTranslation() {
+    const text = document.getElementById('text').value;
+    const languageCode = document.getElementById('language').value;
+
+    const resultContainer = document.getElementById('result');
+    resultContainer.innerText = 'Loading...';
+
+    const params = new URLSearchParams();
+    params.append('text', text);
+    params.append('languageCode', languageCode);
+
+    fetch('/translate', {
+      method: 'POST',
+      body: params
+    }).then(response => response.text())
+    .then((translatedMessage) => {
+      resultContainer.innerText = translatedMessage;
+    });
+  }
+*/
+
+// time bottom for fetch
+async function showServerTime(event) {
+    event.preventDefault();
+
+    const data = new FormData(event.target);
+    const value = Object.fromEntries(data.entries());
+
+
+    const responseFromServer = await fetch('/hello' , {
```

Comment:
/hello(which directs to HelloWorldServlet) doesn't seem to respond server time? Make sure to either correct the servlet behavior or change the function name to what it is actually doing.

Reply:
Yes, it doesn't response server time, I will change the name.